stubs/simple-websocket/simple_websocket/aiows.pyi

```diff
@@ -0,0 +1,129 @@
+import asyncio
+import socket
+from _typeshed import Incomplete, Unused
+from _typeshed.wsgi import WSGIEnvironment
+from collections.abc import Awaitable, Callable
+from ssl import SSLContext
+from typing import Any, Literal, TypedDict, type_check_only
+
+from wsproto import ConnectionType, WSConnection
+from wsproto.events import Request
+from wsproto.frame_protocol import CloseReason
+
+from .asgi import WebSocketASGI, _SocketDataBase, _SocketDataBytes, _SocketDataProtocol, _SocketDataStr
+
+class AioBase:
+    subprotocol: str | None
+    connection_type: ConnectionType
+    receive_bytes: int
+    ping_interval: float | None
+    max_message_size: int | None
+    pong_received: bool
+    input_buffer: list[bytes | str]
+    incoming_message: bytes | str | None
+    incoming_message_len: int
+    connected: bool
+    is_server: bool
+    close_reason: CloseReason
+    close_message: str
+    rsock: asyncio.StreamReader
+    wsock: asyncio.StreamWriter
+    event: asyncio.Event
+    ws: WSConnection | None
+    task: asyncio.Task[None]
+    def __init__(
+        self,
+        connection_type: ConnectionType | None = None,
+        receive_bytes: int = 4096,
+        ping_interval: float | None = None,
+        max_message_size: int | None = None,
+    ) -> None: ...
+    async def connect(self) -> None: ...
+    async def handshake(self) -> None: ...
+    async def send(self, data: bytes | Any) -> None: ...
+    async def receive(self, timeout: float | None = None) -> bytes | str | Any: ...
+    async def close(self, reason: CloseReason | None = None, message: str | None = None) -> None: ...
+    def choose_subprotocol(self, request: Request) -> str | None: ...
+
+@type_check_only
+class _AioServerRequest(TypedDict):
+    # this is `aiohttp.web.Request`
+    aiohttp: Incomplete
+    sock: None
+    headers: None
+
+class AioServer(AioBase):
+    request: _AioServerRequest
+    headers: dict[str, Any]
```

Comment:
Here we don't require it, because `dict[str, Any]` is a very common idiom and obvious.